src/views/Project/index.jsx

```diff
@@ -337,35 +384,62 @@ export default class Project extends Component {
                 ...(bug.keywords || []),
                 ...extractWhiteboardTags(bug.whiteboard),
               ],
-              summary: bug.summary,
+              summary: {
+                title: bug.summary,
+                url: `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}`,
+              },
               lastUpdated: bug.lastChanged,
               id: bug.id,
-              url: `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}`,
             })),
-          'summary'
+          'summary.title'
         )) ||
       [];
+
+    Object.keys(pageCursors.github).forEach(x => {
+      hasNextPage = !hasNextPage ? pageCursors.github[x].hasNextPage : true;
+    });
+
+    if (!hasNextPage) {
+      Object.keys(pageCursors.bzGoodFirst).forEach(x => {
+        hasNextPage = !hasNextPage
+          ? pageCursors.bzGoodFirst[x].hasNextPage
+          : true;
+      });
+    }
+
+    if (!hasNextPage) {
```

Comment:
What do you think about replacing all the if block of `hasNextPage` to
```suggestion
    const hasNextPage =
      Object.keys(pageCursors.github).some(
        x => pageCursors.github[x].hasNextPage
      ) ||
      Object.keys(pageCursors.bzGoodFirst).some(
        x => pageCursors.bzGoodFirst[x].hasNextPage
      ) ||
      Object.keys(pageCursors.bzMentored).some(
        x => pageCursors.bzMentored[x].hasNextPage
      );
```